src/molecules/feature-list/index.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from "react";
+import styled from "styled-components";
+
+export interface FeatureProps {
+
+}
+
+const StyledFeatureList = styled.div`
+	max-width: 1080px;
+	margin: 0 auto;
+	vertical-align: top;
+`;
+
+
+const FeatureList: React.StatelessComponent<FeatureProps> = (props): JSX.Element => {
```

Comment:
When you remove the empty interface you can type your stateless component like this:
1. `const FeatureList: React.StatelessComponent<{}>`
or
2. `const FeatureList: React.StatelessComponent<void>`